### PR TITLE
Some improvements for Crowdin Integration page for issue #724

### PR DESCRIPTION
**Pull request by duongdo27**
@dogi 
This is my [rawgit link](https://rawgit.com/duongdo27/open-learning-duongdo.github.io/fix_crowdin_integration_page/#!pages/crowdinintegration.md).

### CODE REVIEW

File: pages/crowdinintegration.md [hunk 1]
```diff
@@ -57,7 +60,7 @@ To see examples of screenshots, click on a tagged string, and take a look at the
 >When you see an untagged string, it might be helpful to ask yourself where can you find the string in the app? For example, where do think the string “compose” can be found? The most likely place is in the email page. Go there and take a screenshot if it is not already tagged. 
 
 ###Step 2: Upload and Tag Screenshots:
-####1- Uploading Screenshots:
+####2.1- Uploading Screenshots:
```
Comment (mappuji) [inline pages/crowdinintegration.md]: Additional space after dash is needed

File: pages/crowdinintegration.md [hunk 2]
```diff
@@ -79,7 +82,7 @@ To see examples of screenshots, click on a tagged string, and take a look at the
 
 ![New Screenshots](uploads/images/new_screenshot.png)
 
-####2- Tagging Screenshots:
+####2.2- Tagging Screenshots:
```
Comment (mappuji) [inline pages/crowdinintegration.md]: Same, an additional space after dash and before the number `2` is needed.

**Comment by duongdo27** [comment]
@mappuji I added the additional space as suggested. Please let me know if you have any other suggestions.

**Comment by duongdo27** [comment]
@mappuji  Sorry, I am not fully understand your comment. 
In the first box, should I remove the small dot before 1 and 2?
For the second box, it shows up correctly in the rawgit but not in the preview. Should I fix all the sections so the preview is correct?

File: pages/crowdinintegration.md [hunk 3]
```diff
@@ -1,9 +1,12 @@
...
-##Objectives:
+
+## Objectives:
+
 There are two objectives for this assignment:
-* 1- Provide exactly one screenshot for the strings in BeLL app, and tag it.
-* 2- Write good description of the strings in Context.
+
+* 1 - Provide exactly one screenshot for the strings in BeLL app, and tag it.
+* 2 - Write good description of the strings in Context.
```
Comment (paulbert) [inline pages/crowdinintegration.md]: I think just using an ordered list here would be more readable.  [Check out the markdown documentation](https://daringfireball.net/projects/markdown/syntax#list) if you need more info.

**Comment by duongdo27** [comment]
@paulbert I just change it to ordered list.

File: pages/crowdinintegration.md [hunk 4]
```diff
@@ -57,7 +60,7 @@ To see examples of screenshots, click on a tagged string, and take a look at the
 >When you see an untagged string, it might be helpful to ask yourself where can you find the string in the app? For example, where do think the string “compose” can be found? The most likely place is in the email page. Go there and take a screenshot if it is not already tagged. 
 
 ###Step 2: Upload and Tag Screenshots:
-####1- Uploading Screenshots:
+####2.1 - Uploading Screenshots:
```
Comment (mappuji) [inline pages/crowdinintegration.md]: The header is not properly formatted

You need to add space after the last hashtag.

change
`####1- Uploading Screenshots:` to `#### 1- Uploading Screenshots:`
`####2.1 - Uploading Screenshots:` to `#### 2.1 - Uploading Screenshots:`

File: pages/crowdinintegration.md [hunk 5]
```diff
@@ -1,9 +1,12 @@
...
 # Crowdin Integration
-##Objectives:
+
+## Objectives:
+
 There are two objectives for this assignment:
-* 1- Provide exactly one screenshot for the strings in BeLL app, and tag it.
-* 2- Write good description of the strings in Context.
+
+1. Provide exactly one screenshot for the strings in BeLL app, and tag it.
```
Comment (mappuji) [inline pages/crowdinintegration.md]: Awesome

File: pages/crowdinintegration.md [hunk 6]
```diff
@@ -79,7 +82,7 @@ To see examples of screenshots, click on a tagged string, and take a look at the
 
 ![New Screenshots](uploads/images/new_screenshot.png)
 
-####2- Tagging Screenshots:
+####2.2 - Tagging Screenshots:
```
Comment (mappuji) [inline pages/crowdinintegration.md]: Same problem, improper header

**Comment by duongdo27** [comment]
Hi @mappuji 
I add some spaces and blank lines to fix the problem.
Here is the [preview](https://github.com/duongdo27/open-learning-duongdo.github.io/blob/fix_crowdin_integration_page/pages/crowdinintegration.md) and [my rawgit](https://rawgit.com/duongdo27/open-learning-duongdo.github.io/fix_crowdin_integration_page/#!pages/crowdinintegration.md). Thank you.

**Comment by duongdo27** [comment]
Thank you so much, @mappuji

**Pull request merged**